changeset:   6240:77473bfeaa7a
tag:         tip
user:        Panu Matilainen <pmatilai@redhat.com>
date:        Fri Jan 25 17:33:35 2008 +0200
files:       installplatform
description:
Urgh, add geode to x86_64 secondary arch subst-crack...
(transplanted from 1196f54f1c475819b16c419f3f060c9fe3ff1e91)


diff -r 8e1c3b69d2af -r 77473bfeaa7a installplatform
--- a/installplatform	Fri Jan 25 14:08:27 2008 +0200
+++ b/installplatform	Fri Jan 25 17:33:35 2008 +0200
@@ -36,7 +36,7 @@ case "$arch" in
   sparc*) SUBSTS='s_sparc\(64\|64v\|v9v\|v9\)_sparc_ s_sparc64_sparcv9_;s_sparc\([^v]\|$\)_sparcv9\1_ s_sparcv9_sparc64_;s_sparc\([^6]\|$\)_sparc64\1_' ;;
   powerpc*|ppc*) SUBSTS='s_ppc64_ppc_ s_ppc\([^6ip]\|$\)_ppc64\1_ s_ppc\([^6ip]\|$\)_ppciseries_ s_ppc\([^6ip]\|$\)_ppcpseries_ s_ppc\([^6ip]\|$\)_ppc64iseries_ s_ppc\([^6ip]\|$\)_ppc64pseries_' ;;
   s390*) SUBSTS='s_s390x_s390_ s_s390\([^x]\|$\)_s390x\1_' ;;
-  x86_64|amd64|ia32e) SUBSTS='s,x86_64,x86_64, s,x86_64,ia32e, s,x86_64,amd64, s,x86_64,i386, s,x86_64,i486, s,x86_64,i586, s,x86_64,i686, s,x86_64,pentium3, s,x86_64,pentium4, s,x86_64,athlon,' ;;
+  x86_64|amd64|ia32e) SUBSTS='s,x86_64,x86_64, s,x86_64,ia32e, s,x86_64,amd64, s,x86_64,i386, s,x86_64,i486, s,x86_64,i586, s,x86_64,i686, s,x86_64,pentium3, s,x86_64,pentium4, s,x86_64,athlon, s,x86_64,geode,' ;;
   *) SUBSTS=y___ ;;
 esac
 

